app/controllers/application_controller.rb

```diff
@@ -49,11 +49,14 @@ def current_session_id
     session.to_h['session_id']
   end
 
-  def strip_empty_checkboxes(form_key, fields)
-    if params[form_key].present?
-      fields.each do |field|
-        params[form_key][field] = params[form_key][field]&.reject(&:blank?) unless params[form_key][field].is_a?(String)
-      end
+  def strip_empty_checkboxes(fields, form_key = nil)
+    params_to_strip = if params[form_key].present?
```

Comment:
one-liner is more readable I believe

Reply:
Rubocop keeps doing this style 😔🤖

Reply:
if you type this:

```
params_to_strip = params[form_key].present? ? params[form_key] : params
```

rubocop complains?

Reply:
Ah no I see what you mean now. Yeah I prefer this too 👍🏼